### DISCUSSION TITLE
Algolia search for Laravel 5.x and 4.x docs is broken (not indexed?)

### DISCUSSION THREAD

**Discussion by HenkPoley** [laravel/framework#53435] [General]
### Laravel Version

11.30.0

### PHP Version

8.3.13

### Database Driver & Version

Microsoft SQL Server 2022 (RTM-CU15-GDR) (KB5046059) - 16.0.4150.1 (X64)

### Description

The docs nor the laravel.com has really any Issue tracker for this. So I'm posting it here.

Sometimes you really want to look up some legacy syntax. When you search inside https://laravel.com/docs/5.0/middleware (or any 5.x/4.x) there are no results:

![image](https://github.com/user-attachments/assets/31a53742-253b-4b08-a7bb-4ff68320a431)

### Steps To Reproduce

Navigate to the an older documentation page: https://laravel.com/docs/5.8/

Use the search bar at the top.

**Comment by AhmedAlaa4611** [comment]
The issue is not related to a specific version of the docs. There are also documented methods in the latest version of the Laravel docs that are not searchable. See: https://github.com/laravel/framework/discussions/55819
